test/internals/routes.js

```diff
@@ -178,12 +178,23 @@ describe('Internals - routes', () => {
       }
     })
 
+    it('should return the root url of the app if not passed a path', async () => {
+      const mock = {
+        path: undefined
+      }
+
+      const output = fullyQualifiedLocalPath(mock.path)
+
+      expect(output).to.be.a.string()
+      expect(output).to.equal(mockAppDomain)
+    })
```

Comment:
Do you really need both tests here - since mockAppDomain is already a string the second test will fail anyway if output is not a string - (Mark)